src/processes/helper/createEmptyChannel.js

```diff
@@ -0,0 +1,28 @@
+const {CHANNEL_TYPE_STRING} = require('../../utils');
+const {LogError} = require('../../databases/models');
+const StreamChat = require('stream-chat').StreamChat;
+
+const createEmptyChannel = async (userId, usersFollowed) => {
+  try {
+    const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
+
+    await Promise.all(
+      usersFollowed.map(async (userFollowed) => {
+        const members = [userId, userFollowed];
+        const channel = await client.channel(CHANNEL_TYPE_STRING.CHAT, {
+          members,
+          created_by_id: userId
+        });
+
+        await channel.create();
+      })
+    );
+  } catch (error) {
+    await LogError.create({
+      message: error.message
+    });
+    console.error('createEmptyChannel: ', error);
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good that you're logging errors when they occur. However, after logging the error, consider rethrowing it or throwing a new error so that the caller of this function can handle it appropriately. This way, the caller can decide what to do in case of an error, such as retrying the operation, logging additional information, or notifying the user.
```diff
-    console.error('createEmptyChannel: ', error);
+    console.error('createEmptyChannel: ', error);
+    throw error;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->